packages/nimble-components/src/select/index.ts

```diff
@@ -837,6 +897,40 @@ export class Select
         this.committedSelectedOption = options[this.selectedIndex];
     }
 
+    private setActiveOption(newActiveIndex: number): void {
+        const selectedIndex = this.openActiveIndex ?? this.selectedIndex;
```

Comment:
How about `previousActiveIndex` instead of `selectedIndex` for this variable name?